### DISCUSSION TITLE
Github education benefits

### DISCUSSION THREAD

**Discussion by tristanrauhut** [community/community#145291] [GitHub Education]
### Hi everyone,

I approved on November 19, but still can not access my benefits.
Glad if someone can help :)

**Comment by boutdatansec** [comment]
My request was also approved on Nov 19th and I couldn't get it yet.  FAQ says to wait up to 5 days when not on busy season,  so we'll probably have it by next week ;)

**Comment by davevad93** [comment]
Hi @tristanrauhut ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @tristanrauhut was approved and your benefits are now available.`

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Reply by tristanrauhut** [reply]
Alright, thank you!

**Reply by davevad93** [reply]
You're welcome @tristanrauhut 👍
